reference/api/search.md

```diff
@@ -432,24 +441,24 @@ To get only the `overview` and `title` fields, set `attributesToRetrieve` to `["
 ### Attributes to crop
 
 **Parameter**: `attributesToCrop`
-**Expected value**: an array of attribute or `["*"]`
+**Expected value**: an array of attributes or `["*"]`
 **Default value**: `null`
 
-Crops the selected attributes' values in the returned results to the length indicated by the [`cropLength`](#crop-length) parameter.
+Crops the selected fields to the length indicated by the [`cropLength`](#crop-length) parameter.
```

Comment:
Good point. I will revise the current text (don't want to immediately accept your suggestion because I'm trying to keep some uniformity on how we present search parameters).

Regarding your observation: completely agree. Let's continue the discussion in the issue.